worldedit-core/src/main/java/com/sk89q/worldedit/command/tool/LongRangeBuildTool.java

```diff
@@ -92,6 +93,7 @@ public boolean actPrimary(Platform server, LocalConfiguration config, Player pla
 
         try (EditSession editSession = session.createEditSession(player, "LongRangeBuildTool")) {
             try {
+                editSession.disableBuffering();
```

Comment:
why's this been added? disableBuffering is quite a bad idea in FAWE unless you know what it does (effectively enables fast mode as it calls disableQueue)